apps/postybirb-ui/src/app/postybirb-layout/drawers/tag-converter-drawer.tsx

```diff
@@ -161,54 +170,56 @@ function WebsiteConverterInputs({
               </tr>
             </thead>
             <tbody>
-              {websites.map((website) => (
-                <tr key={website.id}>
-                  <td
-                    style={{
-                      padding: '8px',
-                      borderBottom: '1px solid #eee',
-                      width: '30%',
-                    }}
-                  >
-                    <Group>
-                      <Text>{website.displayName}</Text>
-                      {value[website.id]?.trim().length > 0 && (
-                        <Badge size="xs" variant="light" color="green">
-                          <Trans>Set</Trans>
-                        </Badge>
-                      )}
-                    </Group>
-                  </td>
-                  <td
-                    style={{
-                      padding: '8px',
-                      borderBottom: '1px solid #eee',
-                      width: '70%',
-                    }}
-                  >
-                    <TextInput
-                      size="xs"
-                      value={value[website.id] ?? ''}
-                      placeholder={_(
-                        msg`Enter conversion for ${website.displayName}`,
-                      )}
-                      onChange={(event) =>
-                        handleChange(website.id, event.currentTarget.value)
-                      }
-                      onBlur={(event) =>
-                        handleBlur(website.id, event.currentTarget.value)
-                      }
-                      disabled={disabled}
-                    />
-                  </td>
-                </tr>
-              ))}
+              {websites.map((website) => {
+                const websiteName = website.displayName;
```

Comment:
for lingui trans macro its better to pass data as variable with name instead of expression